src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveOrganisationsTest.java

```diff
@@ -45,6 +49,9 @@
 @Slf4j
 @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
 class RetrieveOrganisationsTest extends AuthorizationEnabledIntegrationTest {
+    static final String SINCE_TIMESTAMP_FORMAT = "yyyy-MM-dd'T'HH:mm:ss";

```

Comment:
as with the other PR, perhaps since should be last updated since instead, so its obvious what its for